### PR TITLE
Updated CHANGELOG and package.json for 3.14.1 release

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.14.1 (2020-04-24)
+
+- Restored the bold font style to the `sky-headline` CSS class used by components like key info. [#119](https://github.com/blackbaud/skyux-theme/pull/119)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: Is the "used by components like key info" important to include? If not, I think it'd be fine ending at "class."

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md:3]: I added that bit because it was the style change to the key info component where a consumer noticed the issue.

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Also, should `Restored the bold font style` correct? Looking at the related PR, it looks like we changed the font weight. Is that the same thing? Just wondering if this should actually be `Fixed the font weight for the sky-headline...` ... And does the other class we changed need to be called out as well?

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md:3]: In a previous release we changed the font weight at which the browser loads the Blackbaud Sans bold font but didn't change this code to reflect it, which had the effect of some bold text no longer being bold. So by specifying the new font-weight, we restored the bold styling. The other class is brand new and undocumented, so I didn't see the need to call it out.

Comment (blackbaud-johnly) [reply CHANGELOG.md:3]: Nevermind. Looks like this got merged as I was typing up my comment. Probably not worth worrying about anyway.

**Pull request merged**